app/app.go

```diff
@@ -0,0 +1,141 @@
+package app
+
+import (
+	"fmt"
+	"strings"
+
+	"gihub.com/gogillu/user-manager/cli"
+	"gihub.com/gogillu/user-manager/repository"
+	"gihub.com/gogillu/user-manager/user"
+	"gihub.com/gogillu/user-manager/user/enum"
+)
+
+const (
+	DatastoreFilepath = "./data/users.json"
+	Yes               = "y"
+	OptionAdd         = 1
+	OptionList        = 2
+	OptionDelete      = 3
+	OptionSave        = 4
+	OptionExit        = 5
+)
+
+func Start() error {
+	userRepo := repository.NewRepo()
+	err := userRepo.Load(DatastoreFilepath)
+	if err != nil {
+		return err
+	}
+
+	exit := false
+	for !exit {
+		cli.ShowMenu()
+		choice := cli.GetMenuChoice()
+		var err error
+
+		switch choice {
+		case OptionAdd:
+			err = AddUser(userRepo)
+		case OptionList:
+			err = ListUsers(userRepo)
+		case OptionDelete:
+			err = DeleteUser(userRepo)
+		case OptionSave:
+			err = SaveUsers(userRepo)
+		case OptionExit:
+			err = Exit(userRepo)
+			exit = true
+		default:
+			continue
+		}
+
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func getCourses(courses string) ([]enum.Course, error) {
+	coursesList := strings.Fields(courses)
+	var courseList []enum.Course
```

Comment:
variable name `coursesList,courseList` is not justifying the difference, use name like existingList